src/xdocs/config_imports.xml

```diff
@@ -2546,6 +2541,50 @@ class FooBar {
         <source>
 &lt;module name=&quot;UnusedImports&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+import demo.Demo;  // violation, Unused import - demo.Demo
```

Comment:
Remove this import. It is same as Map.
No need to duplicate violations of same type

Reply:
It is different from Map, it is unused import because class is imported from the same package.

Reply:
From imports perspective there is no difference. If imported class or method is not used - it is violation. Package name does not matter at all.

Reply:
done